src/models/guide.cpp

```diff
@@ -19,8 +20,121 @@
 namespace {
 Logger logger(LOG_MAIN, "Guide");
 
+bool evaluateConditionsEnabledFeatures(const QJsonArray& enabledFeatures) {
+  for (QJsonValue enabledFeature : enabledFeatures) {
+    // If the feature doesn't exist, we crash.
+    const Feature* feature = Feature::get(enabledFeature.toString());
+    Q_ASSERT(feature);
```

Comment:
I think an assert is the wrong approach here, we should just treat it as an unsupported guide in this case.